README.md

```diff
@@ -18,15 +18,15 @@
 
 ## Background
 
-Implementations added to this package are tested against various test suites in order to demonstrate interopability.
+Implementations added to this package are tested against various test suites in order to demonstrate interoperability.
 
 ## Security
 
 Please do not commit any sensitive materials such as oauth2 client secret or client secrets used for signing zcaps or HTTP Signature Headers.
 
 ## Install
 
-- Node.js 14+ is required.
+- Node.js 16+ is required.
```

Comment:
Need to update `engines.node` in `package.json` https://github.com/w3c-ccg/vc-api-test-suite-implementations/blob/main/package.json#L68

Reply:
good catch (it was set to 12 actually) updated here: https://github.com/w3c-ccg/vc-api-test-suite-implementations/pull/32/commits/a9d14d7852a3c48d05872c72658d8a5ac3d74170